npm-global-without-sudo.md

```diff
@@ -21,13 +21,28 @@ npm config set prefix "${HOME}/.npm-packages"
 Add the following to your `.bashrc`/`.zshrc`:
 
 ```sh
-NPM_PACKAGES="${HOME}/.npm-packages"
+NPM_PACKAGES="${XDG_DATA_HOME}/npm-packages"
```

Comment:
`XDG_DATA_HOME` isn't defined on macOS. It should fall back to `HOME`.

Reply:
added a commit that addresses this.